app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -16,24 +16,33 @@ $(function() {
     setupAjaxSearch();
     setupTags();
 
+    function getSelectOptions($el) {
+      var base = {
+            tags : $el.data('collection'),
+            width: $el.data('width') || DEFAULT_SELECT_WIDTH,
+          };
+
+      return $.extend({}, base, $el.data('selectOptions') || {});
+    }
+
     function setupTags() {
       $('.select2-tags', container).each(function(i, el) {
-        var model = $(el).data('model'),
-            method = $(el).data('method'),
-            width = $(el).data('width'),
-            selectOptions = {
-              width: width || DEFAULT_SELECT_WIDTH,
-              tags: $(el).data('collection')
-            };
+        var $el = $(el);
```

Comment:
Unexpected var, use let or const instead                         no-var<br>'$' is not defined                                               no-undef